crates/rolldown/tests/rolldown/errors/json_parse_error/issue-3102/artifacts.snap

```diff
@@ -0,0 +1,18 @@
+---
+source: crates/rolldown_testing/src/integration_test.rs
+snapshot_kind: text
+---
+# Errors
+
+## PARSE_ERROR
+
+```text
+[PARSE_ERROR] Error: Expected `}` but found `EOF`
```

Comment:
@sapphi-red 

Does this consider a better a error message as described in #3102?

Reply:
Yes 👍